### DISCUSSION TITLE
请教：X6实现流程图如何限制【开始】【结束】节点只能拖拽一次？

### DISCUSSION THREAD

**Discussion by gyjcode** [antvis/X6#4540] [Q&A]
const start = graph.createNode({
    id: 'start',
    shape: 'rect',
    label: '开始',
    width: 60,
    height: 30,
    attrs: { ...nodeAttrs },
    ports: { ...ports }
  });
stencil 中 load 的上述节点，graph 的 cell:added 事件获取的 cell.id !== 'start' 是随机生成的 uuid ，如何判断当前节点已经拖入画布，不允许重复拖入？或者如何设置元素的id 不使用 uuid 而使用  node 中定义的 id: 'start' ?

**Comment by gyjcode** [comment]
![局部截取_20241226_100838](https://github.com/user-attachments/assets/5a9f9d55-2084-47bf-b30a-ec679f808616)
我看官方示例也是开始节点可以拖拽多次

**Comment by Yeti-xxx** [comment]
第一次拖拽开始节点时标记一个变量，第二次要拖拽开始节点时检查一下这个变量是否为true,为true就return
